app/src/main/java/org/covidwatch/android/presentation/MainActivity.kt

```diff
@@ -45,13 +46,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun notifyUserOfPossibleExposure() {
```

Comment:
Is this function not a duplicate of the functionality in `NotifyAboutPossibleExposureUseCase.kt`?

Reply:
Yes it is, but we might need it to show notification when firebase triggers a change in https://github.com/covid19risk/covidwatch-android/pull/90.